packages/compiler-core/src/transforms/transformElement.ts

```diff
@@ -145,7 +145,8 @@ export const transformElement: NodeTransform = (node, context) => {
     }
 
     // children
-    if (node.children.length > 0) {
+      if (node.children.length > 0) {
+      let filterdChildren = node.children
```

Comment:
nit: should be `filteredChildren`, but I think it should just be named `"children"`.

```suggestion
      let children = node.children
```